guides/release/community-release.md

```diff
@@ -0,0 +1,98 @@
+# GitHub Release Process
+
+## 1. Requirements
```

Comment:
Is there any reason why this header is of `##` level, and the rest of 3 `###`?

Reply:
a very good question :) 
I will remove it!